docs/dependence_metrics.rst

```diff
@@ -63,68 +64,85 @@ For a **TE-dependence model**, if there were no fluctuations in :math:`S_0`:
 
 Note that TE is a parameter in this model. Hence, it is TE-dependent.
 
+Applying our models to signal decay curves
+******************************************
+
 As an example, let us simulate some data.
 We will simulate signal decay for two time points, as well as the signal decay
 for the hypothetical overall average over time.
-In one time point, only S0 will fluctuate.
-In the other, only R2* will fluctuate.
+In one time point, only :math:`S_0` will fluctuate.
+In the other, only :math:`T_2^*` will fluctuate.
 
-.. note::
+.. caution::
```

Comment:
Each individual choice seems reasonable, but it's semi-arbitrary what's a 'note' vs 'tip' vs 'caution.' This isn't a reason to hold up this PR, but perhaps we'll later want to add a brief comment in the developer guide listing each one we're using in the docs & setting up a guideline for which to use.